waitsensor/waitsensor.go

```diff
@@ -312,37 +319,26 @@ func (cs *counter) counts2class(count int) string {
 }
 
 type RingBuffer struct {
-	size     int
-	data     []string
-	current  int
-	full     bool
-	labelMap map[string]int
+	size    int
+	data    []float64
+	current int
+	full    bool
 }
 
-func NewRingBuffer(thresholds []Bin, size int) *RingBuffer {
-	data := make([]string, size)
-	labelMap := make(map[string]int)
-	for _, b := range thresholds {
-		labelMap[b.Label] = 0
-	}
```

Comment:
no longer tracking the the labels, but the actual count numbers, since we care about the mean now, and not the mode